hoomd/trigger.py

```diff
@@ -2,15 +2,45 @@
 # This file is part of the HOOMD-blue project, released under the BSD 3-Clause
 # License.
 
+""" :py:class:`hoomd.trigger` enables users to design time points when and while
+    :py:class:`hoomd.Operations` operates. The composite of triggers takes 
```

Comment:
```suggestion
   `hoomd.Operations` operates. The composite of triggers takes 
```
You don't need to preface links. Sphinx can find the right target without the identifier and this is easier to read.